src/scss/components/_product-detail.scss

```diff
@@ -0,0 +1,188 @@
+.product-detail {
+  background-color: var(--gray);
+  padding-top: 0;
+  padding-bottom: 112px;
+
+  @include breakpoint(sm) {
+    padding-top: 52px;
+  }
+
+  .col-sm-6,
+  .col-sm-4 {
+    @include breakpoint(sm) {
+      padding: 0;
+    }
+  }
+
+  &__images {
+    display: flex;
+    width: 92vw;
+    margin: 4vw;
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+    scroll-snap-type: x mandatory;
+    scroll-behavior: smooth;
+    scrollbar-width: none;
+
+    @include breakpoint(sm) {
+      width: 100%;
+      margin: 0;
+      overflow-x: visible;
+      flex-flow: row wrap;
+      align-items: flex-start;
+      align-content: flex-start;
+      margin-bottom: 40px;
+      gap: 1rem;
+    }
+
+    @include breakpoint(lg) {
+      gap: 2rem;
+    }
+  }
+
+  &__images::-webkit-scrollbar {
+    display: none;
+    scrollbar-width: none;
+  }
+
+  &__images > div {
+    scroll-snap-align: start;
+  }
+
+  &__image {
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    width: 92vw;
```

Comment:
Try using a different `width` for the images, so that you can add a spacing between them. Something like this:
`width: 78vw;` and a `margin-right` only on mobile